trace-sdk/src/test/java/io/bitrise/trace/data/management/formatter/device/DeviceOsDataFormatterTest.java

```diff
@@ -0,0 +1,29 @@
+package io.bitrise.trace.data.management.formatter.device;
+
+import org.junit.Test;
+
+import io.bitrise.trace.data.collector.device.DeviceOsVersionDataCollector;
+import io.bitrise.trace.data.dto.Data;
+import io.bitrise.trace.data.dto.FormattedData;
+import io.bitrise.trace.data.management.formatter.BaseDataFormatterTest;
+import io.bitrise.trace.data.resource.ResourceEntity;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Tests for {@link DeviceOsDataFormatter}.
+ */
+public class DeviceOsDataFormatterTest extends BaseDataFormatterTest {
+
+    final String deviceOs = "11.0.0";
```

Comment:
nit: private visibility is enough for this variable.

nit: any String is perfectly ok as the value, but might worth to use a possible value: right now we get the Build.VERSION.SDK_INT (an int value) as a String, so an example could be 30, or 29, etc.

Reply:
good catch re the version number - will update 😅